### ISSUE TITLE
Clicking on the box in the stack report at dashboard screen gives black screen

### ISSUE THREAD

**Issue by arunkumars08** [openshiftio/openshift.io#3371]
Loosely related to this issue - https://github.com/openshiftio/openshift.io/issues/2328

While click on 'View Stack Report' link works just fine as expected, clicking on one of the cards attempts to open a stack report modal but fails in the middle with a black screen without errors in the console.

**Comment by ldimaggi** [comment]
Are you still seeing this error?

![screenshot from 2018-04-30 14-00-15](https://user-images.githubusercontent.com/642621/39442348-da5883a8-4c7e-11e8-8f6e-4a60e244b2b3.png)

**Comment by bmicklea** [comment]
I'm also unable to recreate. Closing. If this is consistently reproducible please includes steps and reopen.

**Comment by aptmac** [comment]
@arunkumars08 are you still seeing this issue? Because I'm seeing it around half of the time.

Steps to reproduce:
- Visit analyze/dashboard page for a space that has pipelines
- Click any of the three boxes: (security issues, licenses, insights)
- Now, either the appropriate box pops up, or I see a blank dark overlay that I cannot get rid of. After playing with this for a while, I see the overlay about 75% of the time, and the popup the other 25%. 

This happens for me on both Firefox and Chrome.

Here's a gif of it happening today in production:
![analytics-overlay](https://user-images.githubusercontent.com/10425301/39723678-91c9bc10-5214-11e8-8cfb-61f07239f553.gif)

**Comment by sivaavkd** [comment]
@arunkumars08 @aptmac I tried this few times and yet, unable to reproduce.

**Comment by aptmac** [comment]
@sivaavkd yes I'm still able to see this quite frequently. It's still intermittent - I do see the correct screen sometimes. Here's a quick gif where I see the error and then the correct screen back-to-back (not shown: me seeing the bug 4 times previous to this clip)

![stack-report-error](https://user-images.githubusercontent.com/10425301/40843221-45c4691e-657e-11e8-8a19-0ca092f9484a.gif)

**Comment by sivaavkd** [comment]
@aptmac thanks.
@arunkumars08 I removed P0 label since that label is not applicable now.
